index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc">
+      <nav id="navbar">
+        <header>Microverse Trials Documentation</header>
+
+        <a href="#Introduction" class="nav-link">Introduction</a>
+        <a href="#What_you_should_already_know" class="nav-link"
+          >What you should already know</a
+        >
+        <a href="#Javascript_and_Java" class="nav-link">Javascript and Java</a>
+        <a href="#hello_world" class="nav-link">hello world</a>
+        <a href="#Variables" class="nav-link">Variables</a>
+        <a href="#Declaring_variables" class="nav-link">Declaring variables</a>
+        <a href="#variables_scope" class="nav-link">variables scope</a>
+      </nav>
+      <section class="right-sec">
```

Comment:
You will agree with me that the project is an article.
Kindly Use the article tag instead because it is more semantic